docs/sources/setup-grafana/configure-security/configure-authentication/grafana-com/index.md

```diff
@@ -0,0 +1,47 @@
+---
+aliases:
+  - ../../../auth/grafana-com/
+description: Grafana Com Authentication
+title: Configure Grafana Com Authentication
+weight: 500
+---
+
+# Configure Grafana Com authentication
+
+To enable GrafanaCom as your authentication provider. You configure GrafanaCom to generate a client ID and a secret key for you to use.
+
+## Create Grafana Com OAuth keys
+
+To use Grafana Com authentication:
+
+1. Login on to Grafana com
+2. Create an OAuth client by visiting your organization and clicking “OAuth Clients”.
+3. Then click “Add OAuth Client Application”.
+4. Add the URL of your running instance of Grafana.
+5. The next step will give you the client ID and secret.
```

Comment:
i'm not sure what this means. What next step? After they enter the URL, what do they do next? Maybe there's a submit button or a Next button?

Reply:
@chri2547 nice to point this out, I have made it a bit more clear with this commit: 97ed25bf87